Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM rocker/r-ver:latest
+
+RUN Rscript -e 'install.packages("remotes")'
+
+COPY DESCRIPTION /bound/DESCRIPTION
+RUN Rscript -e 'remotes::install_deps("/bound", dependencies = TRUE)'
+
+COPY . /bound
+RUN chmod -R +x /bound
```

Comment:
this is just a brute force way to get things working, not ideal

Reply:
not ideal, but this works for me.

Reply:
Am I missing something, or does this dockerfile not have a CMD or ENTRYPOINT?